frontend/src/metabase/components/SingleSelectListField/SingleSelectListField.tsx

```diff
@@ -20,6 +21,8 @@ import {
 import type { Option, SingleSelectListFieldProps } from "./types";
 import { isValidOptionItem } from "./utils";
 
+const DEBOUNCE_FILTER_TIME = 100;
```

Comment:
It used to be 300 with `waitTimeContext` - what's the reason for change?

Reply:
Maybe I should've applied Chesterton's fence here. I reduced the delay because I find the longer delay very distracting.

I don't think a large delay is needed here because the of options is either a) large enough to to make the component switches to using the backend to do the searching, or b) small enough for react to render changes in the list efficiently (the renders all take <3ms) on my machine.

Happy to keep this change out of the PR though if it seems too random.

Reply:
I think it's probably a good change, but it'd be better to make it holistically for consistency.

> Happy to keep this change out of the PR though if it seems too random.

Your call :)

Reply:
I made the same change to the multi list picker and removed the dependency on the `waitTimeContext`. I don't think having one timeout for every use case makes sense.